wikimedia/wikipedia-ios
# Native Editor - Finalize VoiceOver labels

**Phabricator:** 
https://phabricator.wikimedia.org/T347870#9400922

### Notes
This PR builds off of the work in #4729 to wrap up VoiceOver labels on the new editor views.

### Test Steps
1. Navigate to editor on Staging. Turn on Voice Over.
2. Confirm labels on all buttons in the toolbars and input view make sense.
